content/language/java/build-images.md

```diff
@@ -31,44 +31,6 @@ $ git clone https://github.com/spring-projects/spring-petclinic.git
 $ cd spring-petclinic
 ```
 
-## Test the application without Docker (optional)
```

Comment:
We need to check the Java example; ISTR that there was a step that generated a file on the host, which is later copied into the Dockerfile. Don't know if this was updated later  on, but it may mean that some tools are still needed on the host (if that's the case, we should fix it though!)

Reply:
I ran through the Java guide and it appears to work without running it outside of Docker first. I also double checked at least the first docker run in the Go guide.

The Python and Node guides used to require the installs to build the apps from scratch, but have been updated.

Reply:
And just noticed the Go guide shows examples to run tests outside of Docker. I'll remove those.

Reply:
Thank you SO MUCH for checking! I noticed the PR, and recalled looking at these parts in the past, so would have blamed myself for "not commenting" (better safe than sorry 🥳)